pkg/manifests/manifests_test.go

```diff
@@ -2648,6 +2651,17 @@ ingress:
 		t.Fatal("Alertmanager memory request is not configured correctly:", memoryRequestPtr.String())
 	}
 
+	foundSecrets := 0
+	for _, s := range a.Spec.Secrets {
+		if s == "test-secret" || s == "slack-api-token" {
+			foundSecrets++
+		}
+	}
+
+	if foundSecrets != 2 {
+		t.Fatal("Alertmanager secrets not configured correctly")
```

Comment:
it might be worth logging what secrets were found and what was missing?

Reply:
I updated the test to use `slices` package to compare secrets, does that look good now? :)